frontend/src/core/islands/parse.ts

```diff
@@ -42,6 +41,21 @@ interface MarimoIslandCell {
   idx: number;
 }
 
+function parseIslandEditor(code: string | undefined | null): string {
+  if (!code) {
+    return "";
+  }
+  // Remove the first and last character, which are quotes
+  return code.slice(1, -1);
```

Comment:
should we check `code.startWith('"') && code.endsWith('"')` just to be sure?

Reply:
Sure, I didn't really dig into editor behavior, but it might be best addressed there. Maybe we can pull from value directly making the editor basically a cell?

Reply:
actually, i see whats going on. for consistency, the value is always `JSON.strinfied` (it may not be a string), so i would do:

```ts
function parseIslandEditor(code: string | undefined | null): string {
  if (!code) {
    return "";
  }
  try {
      return JSON.parse(code);
  } catch {
     return code;
  } 
```